bin/reusable.py

```diff
@@ -0,0 +1,49 @@
+import json
```

Comment:
Nit: I am ok with the name reusable, but I would personally go with something like utils.py